.github/workflows/validate.yml

```
@@ -55,3 +55,49 @@ jobs:
         run: |
           pnpm cypress:verify
           pnpm cypress
+  test-mac:
+    name: Test on Mac
+    runs-on: macos-latest
```

This can be collapsed with the above [using a matrix](https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs).

```suggestion
    runs-on: [ubuntu-latest, macos-latest]
```

---

Requires a slightly more complicated syntax, but that seems to be working

---

Actually no it isn't

---

I think the failure has something to do with the caches. I cleaned all caches and re-ran all jobs.

---

Looks like it's running both Cypresses now